src/templates/community-events.ejs

```diff
@@ -11,6 +11,29 @@
            <section id="events">
              <p>Events in which Pyramid contributors and developers participate are listed below. To list your event, <a href="https://github.com/Pylons/trypyramid.com/issues/new">create a new issue on GitHub</a> with sufficient information and artwork.</p>
            <h2>Upcoming</h2>
+                <div class="row event-item">
+                    <div class="col-md-8 col-xs-12">
+                    <h3><a href="https://dragonpy.com/sprints">DragonPy Sprints</a></h3>
+                    <h4>April 20-22, 2020</h4>
+                    <p>We are pleased to announce a coding sprint on Pyramid and other projects under the Pylons Project in Ljubljana, Slovenia, on April 20 to 22, 2020, after the weekend of <a href="https://dragonpy.com/">DragonPy</a>. DragonPy is the first Python conference in Slovenia and will bring together Python developers from the nearby region that do not plan to travel to PyCon US. It includes a weekend of advanced talks & beginner-friendly workshops, followed by three days of sprints.</p>
+                    <p>The sprint is free to attend for everyone. You can join us at the <a href="https://dragonpy.com/sprints">venue</a> or remotely on the <a href="https://webchat.freenode.net/#pyramid">#pyramid IRC channel</a>.</p>
+                    <p>Some people that will join the sprint:</p>
+                    <ul>
+                      <li>A bunch of <a href="https://niteo.co/">Niteo</a> developers: <a href="https://github.com/am-on">@am-on</a>, <a href="https://github.com/dz0ny">@dz0ny</a>, <a href="https://github.com/karantan">@karantan</a> and <a href="https://github.com/zupo">@zupo</a></li>
+                      <li><a href="https://github.com/MatthewWilkes">@MatthewWilkes</a></li>
+                      <li><a href="https://github.com/janlikar">@JanLikar</a></li>
+                      <li><a href="https://github.com/jerneju">@jerneju</a></li>
+                    </ul>
+                    <h4>Venue</h4>
+                    <p>To-be-announced on <a href="https://dragonpy.com/sprints">DragonPy Sprints</a> page.<br></p>
+                    </div>
+                  <div class="col-md-4 col-xs-12">
+                    <a href="https://dragonpy.com/">
+                      <img class="img-responsive" src="img/plone-conference-2019-ferrara-logo.svg" alt="Logo of DragonPy Conference, 18 & 19 April 2020, Ljubljana, Slovenia">
```

Comment:
Needs its own logo, or text. SVG is preferred because it does not need resizing and can be copied as is from orginals to resized directory.

- Originals go here: https://github.com/Pylons/trypyramid.com/tree/master/src/img/logos-original
- Resized go here: https://github.com/Pylons/trypyramid.com/tree/master/src/img

Reply:
You also need to update `src/main.js`. Example:

https://github.com/Pylons/trypyramid.com/commit/b1a852c6a4e28d9c63c9a17fffdfbf6ba804163d#diff-1c90ff38a08209f9ebd4d05d1e43358eR103

After I merge, I will run the dist and deploy. Thank you!

Reply:
🤦‍♂ how did I miss this. Thanks, will fix!

Reply:
Done.